storm-core/src/clj/org/apache/storm/daemon/worker.clj

```diff
@@ -743,7 +736,7 @@
                         (fn [& args]
                           (check-credentials-changed)
                           (if ((:storm-conf worker) TOPOLOGY-BACKPRESSURE-ENABLE)
-                            (check-throttle-changed))))
+                            (topology-backpressure-callback))))
```

Comment:
This feels like the wrong place for this. 
For some reason the backpressure is tacked onto the credentials timer?

I know this was already the way it worked, but maybe there's a better place for it now? I don't think it's good that the value of the `task.credentials.poll.secs` config affects the frequency of backpressure polling.